PythonDZ1/DZ1.2.py

```diff
@@ -0,0 +1,15 @@
+#Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+#Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+#а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+#*Пример:*
+#6 -> 1  4  1
+#24 -> 4  16  4
+#60 -> 10  40  10
+
+
+
+n=int(input('Введите число сделанных журавликов кратное 6:'))
+if n%6==0:
+    print(round(n/2/3), round((n/2/3+n/2/3)*2), round(n/2/3))
```

Comment:
Вместо использования функции `round ` лучше просто использовать оператор целочисленного деления
```suggestion
n = int(input('Введите число сделанных журавликов кратное 6:'))
if n % 6 == 0:
    print(n // 6, n // 6 * 4, n // 6)
```